hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/ArtifactManagement.java

```diff
@@ -63,7 +63,7 @@
      *             if given software module does not exist
      */
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_UPDATE_REPOSITORY)
```

Comment:
Would have expected SpringEvalExpressions.HAS_AUTH_CREATE_REPOSITORY for this method...

Reply:
Good catch